### PR TITLE
Project-1-Setup-and-mobile-version-skeleton/

### PR DESCRIPTION
**Pull request by Terbeche**
Hello, this is my first submission for the "Setup and mobile version skeleton" project ( the first one on the portfolio project)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    <li><a href="#"><img src="images/Icons/Linkedin.png" alt=""></a></li>
+    <li><a href="#"><img src="images/Icons/Angelist.png" alt=""></a></li>
+    <li><a href="#"><img src="images/Icons/Twitter.png" alt=""></a></li>
+    <li><a href="#"><img src="images/Icons/Medium.png" alt=""></a></li>
+  </ul>
+
+  <img id="illustration" src="images/illustration-header-mobile.png" alt="the illustration image">
+</div>
+
+</header>
```
Comment (phelian23) [inline index.html:39]: - [ ] The `<header>` element is not for sectioning content and therefore does not introduce a new section in the [outline](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements). Please change the `<header>` tag here to a `<section>` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    </ul>
+  </nav>
+<header>
+
+    <h1>Hey There.
+  I’m Alex</h1>
+    <h3>I’am a Software Developer</h3>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+  don’t hestiate to contact me.</p>
+
```
Comment (phelian23) [inline index.html:25]: - [ ] Kindly align the text content here to the left, currently, in your design, it is aligned to the center

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Terbeche, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Linters are passing
✅ Descriptive pull request
✅ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+}
+
+ul {
+  list-style-type: none;
+}
```
Comment (KayLemba) [inline css/style.css:37]: Kindly add the following styling to ul class this is for your social media icons 

ul {
    list-style-type: none;
    gap: 20px;
    padding: 1px;
}

Implement all the changes requested and your page will look like this:
Screenshot from 2022-04-21 19-42-00
 This is how your design looks:
Screenshot from 2022-04-21 19-45-13

**Review by KayLemba — CHANGES_REQUESTED** [review]
Hi @Terbeche 
Good job so far working on some of the changes the previous reviewer asked you to change.
There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight 🎉
No linter errors. ✔️
GitHub flow is followed. ✔️
PR has a title and summary. ✔️
Required Changes ♻️
Check the comments under the review.

 Please add a descriptive README file to your project, Feel free to use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by danmainah — APPROVED** [review]
Hi @Terbeche , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me using @danmainah in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**